src/HtmlUtils.tsx

```diff
@@ -639,6 +642,66 @@ export function linkifyAndSanitizeHtml(dirtyHtml: string, options = linkifyMatri
     return sanitizeHtml(linkifyString(dirtyHtml, options), sanitizeHtmlParams);
 }
 
+/**
+ * Turns a given URL into an absolute one.
+ * @param {string} url The URL transform
+ * @returns {string} Absolute URL
+ */
+const getAbsoluteUrl = (() => {
+    // Use a single cached anchor element stored in a closure to avoid having to recreate
+    // a new one on every call, which would affect performance
+    let a: HTMLAnchorElement;
+
+    return (url: string) => {
+        if (!a) {
+            a = document.createElement('a');
+        }
+        a.href = url;
+        return a.href;
+    };
+})();
```

Comment:
Is this not equiv to `new URL(url, window.location)`

Reply:
Errrm, it is. 🤦‍♂️

Fixed, thank you.